src/helpers/send-transaction-helper.ts

```diff
@@ -1,8 +1,8 @@
-import { post } from 'request-promise';
-import type { Biconomy } from '..';
-import { BICONOMY_RESPONSE_CODES, config } from '../config';
-import { logErrorMessage, logMessage } from '../utils';
-import { mexaSdkClientMessenger } from './client-messaging-helper';
```

Comment:
Linting mismatch, otherwise looks fine to me, need to test it out properly before merge.

Reply:
Thanks for pointing it out, i will create prettierrc file to make it not dependent on contributors vscode settings :)